### PR TITLE
Beautify test email

### PR DESCRIPTION
**Pull request by LukasReschke**
screen shot 2017-04-18 at 21 31 35

vs.

screen shot 2017-04-18 at 21 31 28

### CODE REVIEW

File: settings/Controller/MailSettingsController.php [hunk 1]
```diff
@@ -142,44 +143,53 @@ public function storeCredentials($mail_smtpname, $mail_smtppassword) {
...
 					'data' => [
 						'message' => (string) $this->l10n->t('A problem occurred while sending the email. Please revise your settings. (Error: %s)', [$e->getMessage()]),
 					],
 					'status' => 'error',
-				];
+				]);
 			}
 
-			return array('data' =>
-				array('message' =>
```
Comment (nickvergessen) [inline settings/Controller/MailSettingsController.php]: I fixed this in the other PR today #4372

File: settings/Controller/MailSettingsController.php [hunk 2]
```diff
@@ -142,44 +143,53 @@ public function storeCredentials($mail_smtpname, $mail_smtppassword) {
...
+
+				$template = $this->mailer->createEMailTemplate();
+				$template->addHeader();
+				$template->addHeading($this->l10n->t('Well done, %s!', [$displayName]));
+				$template->addBodyText($this->l10n->t('If you received this email, the email configuration seems to be correct.'));
+				$template->addFooter();
+
 				$message = $this->mailer->createMessage();
-				$message->setTo([$email => $this->userSession->getUser()->getDisplayName()]);
-				$message->setFrom([$this->defaultMailAddress]);
```
Comment (ChristophWurst) [inline settings/Controller/MailSettingsController.php]: `from` doesn't seem to be set now – was that intended? 😉

Comment (LukasReschke) [inline settings/Controller/MailSettingsController.php]: Yes.

Comment (LukasReschke) [inline settings/Controller/MailSettingsController.php]: See also https://github.com/nextcloud/server/pull/4380 :)

Comment (ChristophWurst) [inline settings/Controller/MailSettingsController.php]: Makes sense 👍 thanks

**Review by ChristophWurst — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Let me rebase and fix the conflicts.

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Pull request merged**